packages/extension-browser/src/devtools/views/pages/config/config-section.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+
+import * as styles from './config-section.css';
+
+type Props = {
+    children: any;
+    title: string;
+}
+
+/**
+ * Wrap related configuration items with a common container and styles.
+ */
+const ConfigSection = ({ children, title }: Props) => {
+    return (
+        <section className={styles.root}>
+            <h1 className={styles.header}>
```

Comment:
We are going to have more than one `h1`, are we ok with that?

Reply:
Yep, multiple `h1`s were introduced in HTML5 so long as there's only one `h1` per "sectioning" element. 

Nesting is allowed too so a nested `section` could have it's own `h1` and that's also fine. Browsers will style `h1`'s in nested `section`s as `h2`, `h3`, etc. according to the nesting level.

Reply:
ok :)